src/task/MonitorTask.ts

```diff
@@ -396,6 +396,7 @@ export default class MonitorTask
     presentAttendeeId: string,
     present: boolean
   ): void => {
+    this.logger.info('Triggering attendeePresenceReceived event');
```

Comment:
Why not

```
this.logger.info(`attendeePresenceReceived: ${this.context.meetingSessionConfiguration.credentials.attendeeId}`);
```
?

Reply:
Updated: 
this.logger.info(`attendeePresenceReceived: ${attendeeId}`);